### DISCUSSION TITLE
Website Feedback

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#6] [General]
Please drop your feedback

**Comment by lizarragadev** [comment]
Hi everyone, I'm Gus from Bolivia 🇧🇴 and I'm happy to help!

I found some small details on the website:
In the **Spanish** section:

### - In the language section
When we are in Spanish and we press to change the language, it says the word: **"Mudar"** and I think that the correct word would be: **"Cambiar"**. And the word "English" is in lowercase.

<img width="248" alt="Screen Shot 2022-06-28 at 22 04 17" src="https://user-images.githubusercontent.com/1924390/176335299-a09b8762-c23d-42fd-b7be-3993c5c0f505.png">

### - Button: _"quiero ser un orador"_
The letter Q is lowercase, in other languages it's in camelcase.

![Screen Shot 2022-06-28 at 21 55 14](https://user-images.githubusercontent.com/1924390/176334359-ccba35f5-ce84-4411-a185-647b96050f43.png)

### - In the section: _"Los temas incluirán:"_ 
I consider that the correct translation would be only **_"Seguridad"_** without the _"La"_ prefix.

![Screen Shot 2022-06-28 at 21 58 33](https://user-images.githubusercontent.com/1924390/176334706-c029e145-63c3-4eae-92a5-71d0f92056c5.png)

### - The section title: "nuestros oradores" 
In English is in Camelcase, but in Spanish it's in lowercase.
![Screen Shot 2022-06-28 at 21 59 30](https://user-images.githubusercontent.com/1924390/176334811-bd7d5549-f901-40be-98af-3159ab1fe856.png)

![Screen Shot 2022-06-28 at 21 59 47](https://user-images.githubusercontent.com/1924390/176334844-49f521eb-1f2c-485e-bf71-0b8ca0ee3bd2.png)

### - In the section: "Únete a la conferencia global de tecnología Latinx"
In the paragraph below it says: "technologia" but it must be "tecnología" a small mistake.

![Screen Shot 2022-06-28 at 22 02 41](https://user-images.githubusercontent.com/1924390/176335131-333c8c27-7464-4185-9b89-72e4818e18d9.png)

### - When adding the session to Google Calendar
It doesn't show the details, perhaps because the details of the event have not yet been closed.

![Screen Shot 2022-06-28 at 22 03 10](https://user-images.githubusercontent.com/1924390/176335181-e8f09b8e-c610-48e2-b1e9-48d2027cec8f.png)


And that's all I could see, if I see anything else I'll write it here. ☺
